static/chrome/Dockerfile

```diff
@@ -22,6 +22,8 @@ COPY entrypoint.sh /
 RUN \
     chmod +x /usr/bin/chromedriver && \
     chown -R selenium:nogroup /etc/opt/chrome/policies/managed/
+
+RUN echo "cookie-file = ~/.config/pulse/cookie" >> /etc/pulse/client.conf
```

Comment:
Let's remove changes under `static/chrome` as they are not related to video recording and seem to duplicate things in #646.

Reply:
Yes sure.  It was based not on master. Now rebased it and force pushed.